docs/platforms/javascript/common/best-practices/shared-environments.mdx

```diff
@@ -104,3 +104,30 @@ If you notice that Sentry is not capturing error events from the browser extensi
 You can disable that by going to your **Project Settings > Inbound Filters** and disabling filtering out errors known to be caused by browser extensions.
 
 Read more about Inbound Filters in the product documentation under [Inbound filters](/concepts/data-management/filtering/#inbound-data-filters).
+
+## Skipping the Browser Extension Check
+
+_Available in all browser-based SDKs since version `8.37.0`_
+
+If for some reason, the SDK wrongfully detects that it's initialized in a browser extension, you can skip the check by specifying the `skipBrowserExtensionCheck` option when initializing the SDK:
+
+```javascript
+import * as Sentry from "@sentry/browser";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  skipBrowserExtensionCheck: true,
+  // ...
+});
+```
+
+<Alert level="danger">
+
+Don't use this option light-hearted if you're in fact using the SDK in a browser extension or another shared environment.
+Initializing the SDK via `Sentry.init` has no advantages over manually setting up the client and scope as described on this page.
+You'd risk quota increases with unactionable issues, interference with other Sentry SDKs and data leakage by doing so.
+
+</Alert>
+
+This option is purely meant as an escape hatch if our browser extension check is incorrectly detecting a browser extension.
+An example for this might be a
```

Comment:
I think there is something missing

Reply:
lol good catch, thx :) Finished the sentence now